keyboards/snes_macropad/snes_macropad.c

```diff
@@ -58,7 +60,8 @@ static void oled_render_keylog(void) {
 }
 
 __attribute__((weak)) const char * get_layer_name_user(int layer) {
-    return "Unknown";
+    snprintf(layer_str, sizeof(layer_str), "%d", layer);
+    return layer_str;
```

Comment:
Assuming the current use case of  `get_layer_name_user`,
```suggestion
    return get_u8_str(layer_str, ' ');
```